api/get_map.js

```diff
@@ -109,5 +109,6 @@ function refs2nodes (doc) {
       element[prop] = doc.element[prop]
     }
   }
+  element.version = doc.version
```

Comment:
unclear if this will be still necessary after the flattening work @noffle ?

Reply:
It won't! This is happening I believe because kappa-osm is still using the

```js
{
  type: 'osm/element',
  id: '17632025771769724627',
  element: 
   { type: 'node',
     changeset: '6203884536870763866',
     lat: 64.96,
     lon: -121.96,
     timestamp: '2019-03-29T20:37:58.978Z' },
  links: [],
  version: 'd468c90208357ca86bbcfe48c531dc12dfa285cb9f9012c715b5759273635dad@97'
}
```
structure. After the flattening work, it'll just be
```js
{
  id: '17632025771769724627',
  type: 'node',
  changeset: '6203884536870763866',
  lat: 64.96,
  lon: -121.96,
  timestamp: '2019-03-29T20:37:58.978Z' },
  links: [],
  version: 'd468c90208357ca86bbcfe48c531dc12dfa285cb9f9012c715b5759273635dad@97'
}
```

Reply:
great okay, so we can close this PR once we have that flattened, but I'll keep this branch alive (and not delete it) because I'm testing mapeo desktop with it locally

Reply:
I removed the change and kept the tests so we could still merge this after #43